examples/nats/go.mod

```diff
@@ -0,0 +1,10 @@
+module nats-example
+
+go 1.14
+
+
+require (
+  github.com/reugn/go-streams/nats v0.0.0
+)
+
+replace github.com/reugn/go-streams/nats => ../../nats
```

Comment:
It's better to require the NATS connector in the go.mod file under the examples folder with all others.

Reply:
Nice one, I missed those at the examples folder root.